app/javascript/react/components/ReviewErrorList.js

```diff
@@ -1,26 +1,37 @@
 import React from "react"
 import _ from "lodash"
 
-const ErrorList = props => {
+const ReviewErrorList = props => { 
+
   const errantFields = Object.keys(props.errors)
   if (errantFields.length > 0) {
     let index = 0
     const listItems = errantFields.map(field => {
       index++
       return (
+        <div>
         <li key={index}>
           {_.capitalize(field)} {props.errors[field]}
         </li>
+        </div>
```

Comment:
Is this new `div` serving a purpose?  If so we will need to add the `key` to the `div` as that is the top level element being returned within `.map`.  Then we can remove it from the `li` and indent in 13-15.

If its not serving a purpose we can just remove it